tests/aws/services/kms/test_kms.snapshot.json

```diff
@@ -1,6 +1,6 @@
 {
   "tests/aws/services/kms/test_kms.py::TestKMS::test_generate_random_invalid_number_of_bytes[None]": {
-    "recorded-date": "13-04-2023, 11:29:52",
+    "recorded-date": "24-01-2024, 10:43:52",
```

Comment:
We can skip updating existing snapshots for the scope of this PR. 

You can refer to know more about parity testing in our [guidelines](https://docs.localstack.cloud/contributing/parity-testing/).

Reply:
Done - dropped the snapshot changes for all existing tests